vision/unit_tests/module_in_frame/test_modulecheck.py

```diff
@@ -0,0 +1,33 @@
+"""
+This file is used for testing ModuleInFrame
+"""
+
+import os, sys
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
```

Comment:
Lets move the test_moduleckeck file directly to the unit_tests directory and remove the module_in_frame folder here. You will be able to move this second sys.path.insert then.